xclim/checks.py

```diff
@@ -5,17 +5,15 @@
 import numpy as np
 import pandas as pd
 
-"""
-Dev notes
----------
 
-I suggest we use `check` for weak checking, and `assert` for strong checking.
-Weak checking would log problems in a log, while strong checking would raise an error.
-
-`functools.wraps` is used to copy the docstring and the function's original name from the source
-function to the decorated function. This allows sphinx to correctly find and document
-functions.
-"""
+# Dev notes
```

Comment:
Docstrings should explicitly be found within functions or else code evaluation will think that they're uncallable function-like snippets of code. ReST syntax doesn't work here anyway. Comments (#) are better.